utils/augmentation.py

```diff
@@ -26,7 +26,11 @@ def compose_transforms(params, dataset):
             lst_trans.append(RandomRotationTarget(limit=params['training']['augmentation']['rotate_limit'],
                                                   prob=params['training']['augmentation']['rotate_prob']))
 
-    lst_trans.append(ToTensorTarget())
+    if params['training']['normalization']['mean'] and params['training']['normalization']['std']:
+        lst_trans.append(Normalize(mean=params['training']['normalization']['mean'],
+                                   std=params['training']['normalization']['std']))
```

Comment:
This will change slightly in light of my upcoming PR, since it will include Scaling. Should we force scaling to 0-1 range if we're normalizing?

Reply:
Am still up for setting the scale dynamically. Sometimes scalling from [-1-1] may be beneficial.